src/main/resources/data-repository-openapi.yaml

```diff
@@ -340,30 +300,27 @@ definitions:
 
   ## Identifiers ##
 
-  DatasetNameModel:
+  ObjectName:
```

Comment:
Did you mean to omit the Model suffix? I think @thathert is using a maxLength of 63 in the liquibase schema.

Reply:
Yes, we should make maxLength 63 here.

re: names, I'd prefer to use different names for properties vs. objects here. We can use the suffix Model for objects that become classes in the generated output from codegen. ObjectName is a property of one of these objects, and so I'd like to differentiate. If it is useful to also use a suffix for all of these, then I suggest we suffix with `Property`

Reply:
I'm OK with that. Property is overloaded, but so is every other word I can think of! :)